v/vivy.md

```diff
@@ -0,0 +1,8 @@
+# Vivy
+
+![alt text](vivy.jpg) 
+
+- This is her picture when she did the song Tsy hivadika hodory.
```

Comment:
`This photo was taken during her video recording for the song *Tsy hivadika hodory*`